app/data_model/list_store.py

```diff
@@ -40,6 +40,10 @@ def serialise(self):
     def __repr__(self):
         return f"<ListModel name={self.name} items={self.items}, primary_person={self.primary_person}>"
 
+    @property
+    def first(self):
+        return self.items[0]
```

Comment:
This could throw a `KeyError` if the list is empty, we should probably handle it so it fails gracefully, and we can know from logs what the issue was.

Reply:
@MebinAbraham I agree we should handle this gracefully, but do you mean handle `IndexError`?

Reply:
Whoops sorry, I mean't `IndexError` for the list.